### ISSUE TITLE
[Refactor]

### ISSUE THREAD

**Issue by pvaiko** [Courseplay/Courseplay_FS22#3029]
**What is the problem this feature/enhancement solves?**
Side offset for waypoints and tools are positive to the right, negative to the left. This is the opposite of the game's right handed coordinate system, where, looking from above, the x axis points to the left, so a left-side offset is positive, a right side offset is negative.

**Describe the solution you'd like**
Use the game's system everywhere, like course offset and tool offset.

**Comment by rohne83** [comment]
One thing to think about: might be mathematically more correct, but less user friendly. Right now the Hud is intuitive, when you want a larger offset to the left, you click on the minus button left of the number for the working width etc.

**Comment by pvaiko** [comment]
Good point, made it clear in the title that we are talking here about the internal representation of the offset, not the presentation.
